Makefile

```diff
@@ -23,6 +23,9 @@ fmt:
 test:
 	go test -coverprofile coverage.out -v ./...
 
+lint:
+	golangci-lint run
+
```

Comment:
This will fail on the ci pipeline as golang-lint would not be found.
You can install it as separate make target as I did in another project: https://github.com/raffis/kjournal/blob/next/Makefile#L59-L66

Reply:
I'm used to CI environments where it's already there 😅 good catch !